threads.php

```diff
@@ -170,7 +175,7 @@ function cfth_timeline_content($term_id) {
 		if ($prev) {
 			$prev_timestamp = strtotime($prev->post_date_gmt);
 			$this_timestamp = strtotime($_post->post_date_gmt);
-			$prev->threads_data['time_offset'] = $this_timestamp - $prev_timestamp;
+			$prev->threads_data['time_offset'] = ( isset( $term_meta['order_term_meta'] ) && 'DESC' === $term_meta['order_term_meta']) ? $prev_timestamp - $this_timestamp : $this_timestamp - $prev_timestamp;
```

Comment:
This is a confusing way to handle this logic. I would separate the steps and include some documentation to explain _why_ this is being handled in this way. That will make the code easier to maintain long-term.